apps/src/weblab/CdoBramble.js

```diff
@@ -36,6 +36,23 @@ export default class CdoBramble {
     this.lastSyncedVersionId = null;
   }
 
+  getInterface() {
```

Comment:
Where does this get used?

Reply:
just in brambleHost.js (it's hard to tell because i deleted 95% of that file so the github diff is misleading) --

https://github.com/code-dot-org/code-dot-org/blob/f7d349dfc54d6319037dffd94035e27f5402db1c/apps/src/weblab/brambleHost.js#L22-L35

Reply:
ooh thanks, github really tricked me with that diff:
![image](https://user-images.githubusercontent.com/8787187/113783868-468b1b00-96e9-11eb-828b-ab3c70295502.png)